src/main/java/org/codehaus/groovy/control/ResolveVisitor.java

```diff
@@ -1467,10 +1472,10 @@ public void visitClass(final ClassNode node) {
         }
 
         ClassNode sn = node.getUnresolvedSuperClass();
-        if (sn != null) resolveOrFail(sn, node, true);
+        if (sn != null) resolveOrFailPlus(sn, node);
```

Comment:
I think `resolveOrFailPlus` isn't a very meaningful name. Agreed the existing parameter is also confusing.

Reply:
I'll make a minor tweak which I think improves it. It is private, so we can tweak further if needed.